src/lib/autoinstall/clients/clone_system.rb

```diff
@@ -151,8 +152,11 @@ def clone_system(options)
             # always clone general
             Yast::ProductControl.clone_modules + ["general"]
           end
+
         Yast::AutoinstClone.Process(target: target)
         begin
+          ::FileUtils.touch(filename)
```

Comment:
Please add a comment why we are doing this (the file might contain sensitive data etc....).